docs/jax.extend.rst

```diff
@@ -3,6 +3,23 @@
 
 .. automodule:: jax.extend
 
+``jax.extend.linear_util``
+--------------------------
+
+.. automodule:: jax.extend.linear_util
+
+.. autosummary::
+  :toctree: _autosummary
+
+  StoreException
+  WrappedFun
+  cache
+  merge_linear_aux
```

Comment:
```suggestion
```

This is a super niche use case, only in custom_jvp/vjp. Maybe we can keep it extra hidden?

Reply:
I left it here because it's used by Oryx